### ISSUE TITLE
TF Version must < 2.0?

### ISSUE THREAD

**Issue by Hongwei-Wei** [google-research/batch_rl#5]
The code cant work in my environment where TF version is the newest(2.2.0).
because the tf.contrib moudle has been removed?

**Comment by agarwl** [comment]
Yeah, I think Dopamine library still uses tensorflow 1.x and the code is only compatible with that for now :/ . @psc-g for visibility about dopamine.

**Comment by psc-g** [comment]
we're working on both a jax version of dopamine and a TF2 compliant version, stay tuned!

**Comment by Hongwei-Wei** [comment]
> we're working on both a jax version of dopamine and a TF2 compliant version, stay tuned!

Thank you very much!

**Comment by agarwl** [comment]
Closing this issue as dopamine now supports JAX agents! The main difference in offline agents is to simply use the  [FixedReplayBuffer](https://github.com/google-research/batch_rl/blob/f9b89f43add68b603ee562ebcf9dffcb6bf6f841/batch_rl/fixed_replay/replay_memory/fixed_replay_buffer.py#L35) (which directly reads data from numpy) instead of the [tensorflow buffer](https://github.com/google-research/batch_rl/blob/f9b89f43add68b603ee562ebcf9dffcb6bf6f841/batch_rl/fixed_replay/replay_memory/fixed_replay_buffer.py#L136). For an example, see the implementation of [QR-DQN in JAX](https://github.com/google/dopamine/blob/master/dopamine/jax/agents/quantile/quantile_agent.py#L217).
